src/write-archive/index.ts

```diff
@@ -75,6 +83,54 @@ export async function writeTestResult(
   }
 }
 
+/**
+ * Takes a DOM snapshot, which is either a `Buffer` or an object in json form, and maps all `src` attributes that are equivalent to
```

Comment:
Nitpick -- I thought it was snapshotting the DOM at first.
```suggestion
 * Accepts a DOM snapshot, which is either a `Buffer` or an object in json form, and maps all `src` attributes that are equivalent to
```

Reply:
Ah! Yeah, I can see how this is unexpected. I'll make that change.